src/calculateRentalCost.js

```diff
@@ -3,8 +3,19 @@
  *
  * @return {number}
  */
+const discountsDays1 = 3;
+const discountsDays2 = 7;
+const discount1 = 20;
+const discount2 = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < discountsDays1) {
+    return days * 40;
+  } else if (days >= discountsDays1 && days < discountsDays2) {
```

Comment:
There's no need for the second condition `days < discountsDays2` in this `else if` statement because if the execution reaches this point, `days` is already guaranteed to be less than `discountsDays2` due to the previous `if` condition.